sklearn/feature_selection/_multivariate_selection.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
Add these lines: https://github.com/srecupe2/scikit-learn/blob/e42fb227b891593520525cf6cb6302f0edfc12eb/sklearn/feature_selection/_univariate_selection.py#L3. Add you and me

Reply:
Do you mean add us as authors in similar line but in the _multivariate_selection.py file? Do i also have to include these other authors?

Reply:
Yes just add us

Reply:
https://github.com/srecupe2/scikit-learn/commit/0cca1d0f8c89244981a30c22928271880153b171